python/ngen_cal/src/ngen/cal/strategy.py

```diff
@@ -15,15 +15,16 @@ class Algorithm(str, Enum):
     """Dynamic Dimensioned Search Algorithm
     """
     dds = "dds"
+    pso = "pso"
 
-class Objective(Enum):
+class Objective(str, Enum):
     """Enumeration of supported search algorithms
 
     """
     """Dynamic Dimensioned Search Algorithm
     """
     __func_map__ = {"custom": objectives.custom, 
-                    "kling_gupta": objectives.kling_gupta_efficiency,
+                    "kling_gupta": objectives.kge,
```

Comment:
Strongly believe we shouldn't change this/use the same name... can we instead add
```
"one_minus_kling_gupta": objectives.kge,
```
... and really, also change `kge` to be named `one_minus_kge`?

Alternatively, we could add code that detected a non-zero target whenever PSO was used, and automatically optimize for `target - f(x)` instead, issue a NOTICE, and then re-invert the value when we're outputting the objective scores to the user? This last part would be important if we do "magic" like this.

Reply:
Can we put this in an issue for further enhancements?